File: scratch_check_org.py
import os
from supabase import create_client

url = os.environ.get("VITE_SUPABASE_URL", "")
key = os.environ.get("VITE_SUPABASE_ANON_KEY", "")

# Try to read from .env if missing
if not url or not key:
    with open(".env") as f:
        for line in f:
            if line.startswith("VITE_SUPABASE_URL"):
                url = line.split("=")[1].strip()
            elif line.startswith("VITE_SUPABASE_ANON_KEY"):
                key = line.split("=")[1].strip()

supabase = create_client(url, key)

print("--- ORGANIZATIONS ---")
orgs = supabase.table("organizations").select("*").execute()
for o in orgs.data:
    print(o)

print("\n--- PROFILES ---")
profiles = supabase.table("profiles").select("*").execute()
for p in profiles.data:
    print(p)

print("\n--- ORGANIZATION MEMBERS ---")
members = supabase.table("organization_members").select("*").execute()
for m in members.data:
    print(m)

File: scratch_check_org.mjs
import { createClient } from '@supabase/supabase-js';
import dotenv from 'dotenv';

dotenv.config();

const url = process.env.VITE_SUPABASE_URL;
const key = process.env.VITE_SUPABASE_ANON_KEY;

const supabase = createClient(url, key);

async function run() {
  console.log("--- ORGANIZATIONS ---");
  const { data: orgs } = await supabase.from('organizations').select('*');
  console.log(orgs);

  console.log("\n--- PROFILES ---");
  const { data: profiles } = await supabase.from('profiles').select('*');
  console.log(profiles);

  console.log("\n--- ORGANIZATION MEMBERS ---");
  const { data: members } = await supabase.from('organization_members').select('*');
  console.log(members);
}

run();
